### PR TITLE
[OSPK8-641] Add webhook logic to validate OSBMS count field

### PR DESCRIPTION
**Pull request by abays**

### CODE REVIEW

File: api/shared/labels.go [hunk 1]
```diff
@@ -32,4 +32,10 @@ const (
 	// OpenStackNetConfigReconcileLabel - label set on objects on which change
 	// trigger a reconcile of the osnetconfig
 	OpenStackNetConfigReconcileLabel = "osnetconfig-ref"
+	// OpenStackBaremetalSetAppLabel -
+	OpenStackBaremetalSetAppLabel = "osp-baremetalset"
+	// OwnerControllerNameLabelSelector -
+	OwnerControllerNameLabelSelector string = "osp-director.openstack.org/controller"
+	// OwnerUIDLabelSelector -
+	OwnerUIDLabelSelector string = "osp-director.openstack.org/uid"
```
Comment (abays) [inline api/shared/labels.go:40]: I know this is duplicating what we have in the main `common` package.  Moving all the constants to the API just seems like a lot of work for this initial pass.

**Review by stuggi — COMMENTED** [review]
looks good to me. I think that covers what we discussed. maybe add [OSPK8-641] as prefix for the jira we track it.

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**